deps/build.jl

```diff
@@ -14,7 +14,17 @@ if is_windows()
     end
 end
 
-libhttp_parser = library_dependency("libhttp_parser", aliases=aliases)
+function validate_httpparser(name,handle)
+    try
+        p = Libdl.dlsym(handle, :http_parser_url_init)
```

Comment:
Okay, so it turns out that the latest tagged version uses a very old outdated library (about 2.1 or something). Now that has been fixed on master by updating to 2.6.2. The function I'm using for binary validation works for both 2.6.0 onward. There are no API changes between 2.6 and 2.7; only bugfixes.